docs/landing-page/master.adoc

```diff
@@ -98,7 +98,7 @@ include::topics/templates/document-attributes.adoc[]
       <div class="col-sm-3 runtime">
         <div class="fa icon vertx"></div>
         <p >
-          Learn how to use launcher for Eclipse Vert.x projects
+          Learn how to use the Eclipse Vert.x runtime.
```

Comment:
When I suggested to simplify "Learn how to use the WildFly Swarm runtime" to "Learn how to use WildFly Swarm", I thought it would be obvious that the suggestion also applies to all the other runtimes :-)

Reply:
With the build app from scratch, remote debugging or debug logging chapters, I feel all the runtime guides aspire to be general-purpose documentation. I can see what you're saying, but I don't think it will be obvious to a regular reader.

Reply:
@Ladicek fair point, updating.